modules/learn/pages/security/on-the-wire-security.adoc

```diff
@@ -115,13 +115,12 @@ The following limitations apply to cipher-suite configuration.
 [#tls-3-cipher-suite-limitations]
 === TLS 1.3 Cipher-Suite Limitations
 
-As indicated above, only the _Data_, _Search_, _Index_, _Query_, _Eventing_, _Analytics_, and _Backup_ services can use TLS 1.3 cipher-suites.
-TLS 1.3 cipher-suites _cannot_ be used by the Cluster Manager.
+As indicated above, TLS 1.3 cipher-suites can by used by all services; and by the Cluster Manager, XDCR, and Views.
 
-Of the services that _can_ use TLS 1.3 ciphers, only the _Data_ and _Analytics_ services support the custom-configuration of TLS 1.3 ciphers &#8212; _custom-configuration_ of ciphers means the selection of a subset of available ciphers for a custom cipher-suite list, and the specifying of the selected ciphers in the subset in any order judged appropriate.
+However, only the _Data_ and _Analytics_ services support the custom-configuration of TLS 1.3 ciphers &#8212; _custom-configuration_ of ciphers means the selection of a subset of available ciphers for a custom cipher-suite list, and the specifying of the selected ciphers in the subset in any order judged appropriate.
```

Comment:
I don't know what the fully correct statement here would be, but I do know that ns_server (cluster manager) also supports custom-configuration of tls 1.3 ciphers. So at the very least that needs to be added, but someone else will have to confirm whether others do as well.

Reply:
Agree that cluster manager should be included for custom-configuration of TLS 1.3. 
The golang services,(index, n1ql, backup, eventing) TLS1.3 ciphers suites aren't customizable IIRC. 
@bryandmc we should check with service endpoints too, similar to what was tried for ns_server endpoint, i.e., after configuring the cipher suites you can test what cipher that particular golang service endpoint offers using scripts/test-ciphers.sh in ns_server directory.

Reply:
will do @anuthan -- will report back with results.

Reply:
yeah Timofey was right, after testing, all the go based services seem to have the same limitation of not being able to set custom ciphers for tlsv1.3. So @tonyjhillman the correct version of this needs to add in cluster manager, but the rest is fine. That is to say: index, n1ql, backup, eventing .. etc still don't support setting tlsv1.3 cipher suites.

Reply:
Okay, thanks. I have changed to say: 
However, only _Cluster Manager_ and the _Data_ and _Analytics_ services support the custom-configuration of TLS 1.3 ciphers